pkg/scheduler/framework/plugins/volumebinding/scorer_test.go

```diff
@@ -0,0 +1,312 @@
+						},
+					},
+					50,
+				},
+				{
+					classResourceMap{
+						classHDD: &scheduling.VolumeResource{
+							Requested: 50,
+							Capacity:  100,
+						},
+						classSSD: &scheduling.VolumeResource{
+							Requested: 100,
+							Capacity:  100,
+						},
+					},
+					75,
+				},
+				{
+					classResourceMap{
+						classHDD: &scheduling.VolumeResource{
+							Requested: 100,
+							Capacity:  100,
+						},
+						classSSD: &scheduling.VolumeResource{
+							Requested: 100,
+							Capacity:  100,
+						},
+					},
+					100,
+				},
+			},
+		},
+		{
+			name: "custom shape, multiple classes",
+			shape: helper.FunctionShape{
+				{
+					Utilization: 50,
+					Score:       0,
+				},
+				{
+					Utilization: 80,
+					Score:       30,
+				},
+				{
+					Utilization: 100,
+					Score:       50,
+				},
+			},
+			cases: []scoreCase{
+				{
+					classResourceMap{
+						classHDD: &scheduling.VolumeResource{
+							Requested: 0,
+							Capacity:  100,
+						},
+						classSSD: &scheduling.VolumeResource{
+							Requested: 0,
+							Capacity:  100,
+						},
+					},
+					0,
+				},
+				{
+					classResourceMap{
+						classHDD: &scheduling.VolumeResource{
+							Requested: 0,
+							Capacity:  100,
+						},
+						classSSD: &scheduling.VolumeResource{
+							Requested: 30,
+							Capacity:  100,
+						},
+					},
+					0,
+				},
+				{
+					classResourceMap{
+						classHDD: &scheduling.VolumeResource{
+							Requested: 30,
+							Capacity:  100,
+						},
+						classSSD: &scheduling.VolumeResource{
+							Requested: 30,
+							Capacity:  100,
+						},
+					},
+					0,
+				},
+				{
+					classResourceMap{
+						classHDD: &scheduling.VolumeResource{
+							Requested: 30,
+							Capacity:  100,
+						},
+						classSSD: &scheduling.VolumeResource{
+							Requested: 60,
+							Capacity:  100,
+						},
+					},
+					5,
+				},
+				{
+					classResourceMap{
+						classHDD: &scheduling.VolumeResource{
+							Requested: 50,
+							Capacity:  100,
+						},
+						classSSD: &scheduling.VolumeResource{
+							Requested: 100,
+							Capacity:  100,
+						},
+					},
+					25,
+				},
+				{
+					classResourceMap{
+						classHDD: &scheduling.VolumeResource{
+							Requested: 90,
+							Capacity:  100,
+						},
+						classSSD: &scheduling.VolumeResource{
+							Requested: 90,
+							Capacity:  100,
+						},
+					},
+					40,
+				},
+				{
+					classResourceMap{
+						classHDD: &scheduling.VolumeResource{
+							Requested: 100,
+							Capacity:  100,
+						},
+						classSSD: &scheduling.VolumeResource{
+							Requested: 100,
+							Capacity:  100,
+						},
+					},
+					50,
+				},
+			},
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			f := buildScorerFunction(tt.shape)
+			for i, c := range tt.cases {
+				gotScore := f(c.classResources)
+				if gotScore != c.score {
+					t.Errorf("case %d: expected %d, got %d", i, c.score, gotScore)
```

Comment:
tt.Name will be output in the log, so:

```suggestion
					t.Errorf("Expect %d, but got %d", c.score, gotScore)
```